### DISCUSSION TITLE
Configuration issues

### DISCUSSION THREAD

**Discussion by adorableweasel** [authelia/authelia#8139] [Troubleshooting]
I've tried to setup Authelia using [this guide](https://thehomelab.wiki/books/dns-reverse-proxy/page/setup-authelia-to-work-with-nginx-proxy-manager) but I keep getting errors in my logs and I'm not sure what I'm doing wrong

The only thing I've done differently from that guide is use Portainer for the compose file, but I also tried the guides way of making a docker-compose.yml file in the same dir and it got the same errors

My presumption is that I've mapped something wrong with the volumes in docker compose cause it looks like the config file just isn't being checked as everything it says needs to be done is already in the config file from what I can understand

Here's my logs [thinned_Authelia_logs.txt](https://github.com/user-attachments/files/17586246/thinned_Authelia_logs.txt)

Here's my config file [configuration.txt](https://github.com/user-attachments/files/17586230/configuration.txt)

Here's my compose file [compose.txt](https://github.com/user-attachments/files/17586234/compose.txt)

Tysm!

EDIT - File structure [Selection_001](https://github.com/user-attachments/assets/8a6d86ca-d6f1-4d88-9da2-d7183fe9dd1b)

**Comment by Crowley723** [comment]
That guide is using an older version of a few of the config options. 
I would recommend looking at the authelia website regarding the config.

Specifically [session](https://www.authelia.com/configuration/session/introduction/)
[Reset password](https://www.authelia.com/configuration/identity-validation/reset-password/)
[Storage](https://www.authelia.com/configuration/storage/sqlite/)
and [Server](https://www.authelia.com/configuration/miscellaneous/server/)

If you have any trouble, feel free to ask in the various [support channels](https://www.authelia.com/information/contact/#chat)

**Reply by adorableweasel** [reply]
Thank you! I shall investigate this today and let you know if I figure it out :)

**Reply by adorableweasel** [reply]
So I've gone through what I believe needed to be done, but it looks like the logs are almost identical still

configuration - [configuration.txt](https://github.com/user-attachments/files/17598194/configuration.txt)

logs - [_Authelia_logs.txt](https://github.com/user-attachments/files/17598205/_Authelia_logs.txt)

**Reply by james-d-elliott** [reply]
Is that the full logs or are there more that have been truncated/redacted?

**Reply by adorableweasel** [reply]
try this :)

[_Authelia_logs(1).txt](https://github.com/user-attachments/files/17598409/_Authelia_logs.1.txt)

**Reply by james-d-elliott** [reply]
What does `docker inspect Authelia` show?

**Reply by adorableweasel** [reply]
Here ya go [docker.txt](https://github.com/user-attachments/files/17598742/docker.txt)

**Reply by james-d-elliott** [reply]
it shows `/data/compose/11/config:/config:rw` which means `/data/compose/11/config` is bound to the `/config` path, instead of `/opt/docker/authelia/config`. Do you know why this may be? Are you running `docker compose up -d` from within the `/opt/docker/authelia` dir? Is that dir the location of your docker compose file?

**Reply by james-d-elliott** [reply]
Actually easiest solution here is to change the docker compose volumes:

```yaml
services:
  authelia:
    container_name: Authelia
    image: authelia/authelia
#   image: authelia/authelia:4.31
    volumes:
      - /opt/docker/authelia/config:/config
    ports:
      - 9091:9091
    restart: unless-stopped
    healthcheck:
      disable: true
    environment:
      - TZ=Europe/London
    depends_on:
      - redis
```

**Reply by adorableweasel** [reply]
lol that was a stupid mistake, though it's not fixed the problem

As I said as well, I'm using Portainer to deploy the compose file

But I am also trying the /opt/docker/authelia and the /opt/docker/authelia/config dirs to run the compose files and get the same problems

Here's the logs again :) - [_Authelia_logs(1).txt](https://github.com/user-attachments/files/17600304/_Authelia_logs.1.txt)

**Reply by james-d-elliott** [reply]
Does the inspect show a different bind path now?

**Reply by adorableweasel** [reply]
Ah put the wrong path in AGAIN

It's giving different information in the logs now, let me run through this myself and see what I can find :)

**Reply by james-d-elliott** [reply]
Nice, that's a much better outcome, your config is taking effect now!

**Comment by james-d-elliott** [comment]
It'd also be good to get a screenshot or screenshots of your folder structure. Specifically that `config` folder.

**Reply by adorableweasel** [reply]
Added a photo :)
